helmfile/charts/ingress-nginx-probe-ingress/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- $fullName := include "ingress-nginx-probe-ingress.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- $host := printf "%s.%s" "ingress-nginx" (required "Cluster domain required" .Values.ingress.clusterDomain) -}}
```

Comment:
This should be set via `ingressNginx.subdomain` as we do for the rest of our ingresses.

Preferably this should be put together in the helmfile values file instead, and then you should add placeholders for everything into the chart values file so we actually have all available options.